.storybook/main.js

```diff
@@ -6,7 +6,7 @@ module.exports = {
         'storybook-addon-i18n/register.js',
         'storybook-addon-designs',
         '@storybook/addon-links',
-        '@storybook/addon-docs',
+        // '@storybook/addon-docs',
```

Comment:
remove the comments

Reply:
I want to add it in the future. This plugin installs babel register so gulp will use that instead of ts-node register. After we switch to pnpm, this problem will disappear cause we do not depend on babel register directly.